.github/workflows/end-to-end-test-portals.yml

```diff
@@ -0,0 +1,90 @@
+name: End to end testing for Portals
+
+on:
+  push:
+  pull_request:
+    branches: [main]
+env:
+  PW_ALL_BLOBS_DIR: all-blob-reports
+  PORTALS_SUBDIR: apps/portals
+concurrency:
+  group: '${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}'
+  cancel-in-progress: true
```

Comment:
Limit concurrent e2e workflow runs to one per event per branch

Reply:
This is great!